### PR TITLE
Do not build rxjs from source under Bazel (PATCH-ONLY)

### PR DESCRIPTION
**Pull request by gregmagolan**
Patch-only version of #28720

### CODE REVIEW

**Review by IgorMinar — APPROVED** [review]

File: integration/bazel-schematics/package.json.replace [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "@angular/platform-browser-dynamic": "file:../angular/dist/packages-dist/platform-browser-dynamic",
     "@angular/router": "file:../angular/dist/packages-dist/router",
     "core-js": "^2.5.4",
-    "rxjs": "~6.3.3",
+    "rxjs": "~6.4.0",
```
Comment (benlesh) [inline integration/bazel-schematics/package.json.replace]: RxJS 6.4 requires TypeScript 2.8. Is that going to be a problem at all? I know there were a few issues with Angular-CLI users on older versions of Angular-CLI. cc @IgorMinar

**Comment by benlesh** [comment]
Marking this as needing review, as I want to double-check with @IgorMinar  about the RxJS version and TypeScript requirements.

**Comment by benlesh** [comment]
It also seems to need a rebase.

**Comment by gregmagolan** [comment]
@benlesh I'm seeing as landed on the 7.2.x branch in git: 3eb327b67bb757f226e6ad25fb1e024c861d5c6c

File: integration/bazel-schematics/package.json.replace (hunk 1 above)
Comment (gregmagolan) [reply integration/bazel-schematics/package.json.replace]: The particular change here is just for bazel-schematics... although rxjs was updated to 6.4.0 in angular-cli latest-versions.ts

**Comment by IgorMinar** [comment]
7.2.x branch is on TS3.1+: 
https://github.com/angular/angular/blob/3eb327b67bb757f226e6ad25fb1e024c861d5c6c/packages/compiler-cli/package.json#L27

**Comment by IgorMinar** [comment]
and yes - this change already landed as 3eb327b (as @gregmagolan pointed out)

**Pull request closed without merging**